x-pack/plugins/security_solution/server/search_strategy/security_solution/factory/network/details/helpers.ts

```diff
@@ -16,12 +17,10 @@ import {
 import { toObjectArrayOfStrings } from '../../../../../../common/utils/to_array';
 
 export const getNetworkDetailsAgg = (type: string, networkHit: NetworkHit | {}) => {
-  const firstSeen = getOr(null, `firstSeen.value_as_string`, networkHit);
-  const lastSeen = getOr(null, `lastSeen.value_as_string`, networkHit);
-  const autonomousSystem: AutonomousSystem | null = getOr(
-    null,
-    `as.results.hits.hits[0]._source.${type}.as`,
-    networkHit
+  const autonomousSystem: AutonomousSystem | {} = getOr(
+    {},
+    `${type}.as`,
+    unflattenObject(getOr({}, 'as.results.hits.hits[0].fields', networkHit))
```

Comment:
I think you might want to revert the change of Line 20, as this change is not included in your PR. And the author who changed this line might not want it to be backported to 8.3